### PR TITLE
Oetiker node output apis

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- Fix the test checking individual openapi schemas by checking if the main directory contains a $schema key typical of jsonschemas
- Attempt to fix the node-output-list-api-v0.0.1.yaml jsonschema that does not contain the **required** field that python jsonschema expects
-> Please check this as I just added one required field which might not make sense.

## Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] If you design a new module, add your user to .github/CODEOWNERS

[waffle.io]:https://waffle.io/marketing-assets/documents/waffleio_cheatsheet_v1.pdf?utm_source=blog&utm_medium=cheatsheet-ctabutton&utm_campaign=cheatsheet

### CODE REVIEW

File: apis/shared/schemas/node-output-list-api-v0.0.1.yaml [hunk 1]
```diff
@@ -5,6 +5,11 @@ title: node output list api
 description: nodes using the list representation for the output
   must be able to handle the following requests
 type: object
+required: 
+  # the validator does not appreciate when required is missing here... and
+  # sadly does not throw any meaningful error about it... so for now I put this...
+  - getItemList
+  - getItem
```
Comment (oetiker) [inline apis/shared/schemas/node-output-list-api-v0.0.1.yaml]: or maybe have 
```
required: []
```

File: apis/webserver/v0/node-v0.0.1.yaml [hunk 2]
```diff
@@ -67,18 +67,20 @@ paths:
          content:
             application/json:
               schema:
-                oneOf:
-                  - $ref: '../../shared/schemas/node-output-list-api-v0.0.1.yaml#/properties/getItemList/request'
-                  - $ref: '../../shared/schemas/node-output-list-api-v0.0.1.yaml#/properties/getItem/request'
+                type: object
+                # oneOf:
+                #   - $ref: '../../shared/schemas/node-output-list-api-v0.0.1.yaml#/properties/getItemList/request'
+                #   - $ref: '../../shared/schemas/node-output-list-api-v0.0.1.yaml#/properties/getItem/request'
```
Comment (oetiker) [inline apis/webserver/v0/node-v0.0.1.yaml]: oh ? what is the problem ? does it not accept ref here ?

File: apis/shared/schemas/node-output-list-api-v0.0.1.yaml (hunk 1 above)
Comment (sanderegg) [reply apis/shared/schemas/node-output-list-api-v0.0.1.yaml]: hmm I tried it without success.. but now the test is automatic so you can test it as well.

File: apis/webserver/v0/node-v0.0.1.yaml (hunk 2 above)
Comment (sanderegg) [reply apis/webserver/v0/node-v0.0.1.yaml]: I'm not quite sure.. the validator is not very verbose, or maybe too much with too little valuable information sometimes. At least I can say that is where the error came from.

Comment (oetiker) [reply apis/webserver/v0/node-v0.0.1.yaml]: ok

File: apis/shared/schemas/node-output-list-api-v0.0.1.yaml (hunk 1 above)
Comment (oetiker) [reply apis/shared/schemas/node-output-list-api-v0.0.1.yaml]: ok

**Pull request merged**